### PR TITLE
Added Graphql tests

### PR DESCRIPTION
**Pull request by OliverGeneser**

### CODE REVIEW

**Comment by OliverGeneser** [comment]
@prasadtalasila Here are the two Graphql tests with Apollo :smiley:

File: servers/lib/test/e2e/app.e2e.spec.ts
Comment (prasadtalasila) [inline servers/lib/test/e2e/app.e2e.spec.ts:1]: Please remove the following:

1. import of child_process
2. execSync("test/starttraefik.bash");
3. execSync("test/stoptraefik.bash");

The suggested names for the tests:

1. Should return the directory contents requested with HTTP POST query
2. Should return the filecontent requested with HTTP POST query
3. Should return the directory contents requested with GraphQL query
4. Should return the filecontent requested with GraphQL query

A quick question: how are the environment variables supplied to this test?

Comment (prasadtalasila) [inline servers/lib/test/e2e/app.e2e.spec.ts:1]: When `yarn test -e` is run, the tests fail with the following log:

```sh
$ script/test.bash -e
README.md file created with 'content123'
 FAIL  test/e2e/app.e2e.spec.ts
  ● Test suite failed to run

    Cannot find module 'react' from '../node_modules/@apollo/client/react/context/context.cjs'

    Require stack:
      /home/prasad/git/prasadtalasila/DTaaS/servers/lib/node_modules/@apollo/client/react/context/context.cjs
      /home/prasad/git/prasadtalasila/DTaaS/servers/lib/node_modules/@apollo/client/react/react.cjs
      /home/prasad/git/prasadtalasila/DTaaS/servers/lib/node_modules/@apollo/client/main.cjs
      e2e/app.e2e.spec.ts

      10 |   expectedListDirectoryResponse,
      11 | } from "../testUtil";
    > 12 | import { ApolloClient, InMemoryCache, gql } from "@apollo/client";
         | ^
      13 |
      14 | const client = new ApolloClient({
      15 |   uri: `http://localhost:4001${process.env.APOLLO_PATH}`,

      at Resolver._throwModNotFoundError (../node_modules/jest-resolve/build/resolver.js:427:11)
      at Object.<anonymous> (../node_modules/@apollo/client/react/context/context.cjs:6:13)
      at Object.<anonymous> (../node_modules/@apollo/client/react/react.cjs:6:15)
      at Object.<anonymous> (../node_modules/@apollo/client/main.cjs:6:13)
      at Object.<anonymous> (e2e/app.e2e.spec.ts:12:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        2.552 s
```

**Comment by prasadtalasila** [comment]
@OliverGeneser  Further changes: 

1. Please also delete `test/starttraefik.bash` and `test/stoptraefik.bash` scripts.
2. If you can fix the unit and integration tests, that would be fantastic. Lets finish the end-to-end tests and then we can you can look into these tests.

**Pull request closed without merging**